client/app/components/dashboards/dashboard-widget/Widget.less

```diff
@@ -10,6 +10,7 @@
 
 .widget-wrapper {
   .widget-actions {
+    display: flex;
```

Comment:
This is the actual change. (remove `display: none`  from the button and use `display: flex`)